src/Forwarding_DLL.h

```diff
@@ -1,2 +1,12 @@
-extern "C"  __declspec(dllexport) double* Forwarding(int lx, int ly, int point_count, int prism_count, double x_min, double dx, double x_max,
+#pragma once
+
+#ifdef _MSC_VER
+	#define DLLEXPORT __declspec(dllexport)
+#else
+	#define DLLEXPORT
+#endif
+
+extern "C" {
+	double* DLLEXPORT Forwarding(int lx, int ly, int point_count, int prism_count, double x_min, double dx, double x_max,

```

Comment:
The revised codes could not be built successfully. We edited here, put "DLLEXPORT" first.